drivers/serial/uart_npcx.c

```diff
@@ -17,6 +17,7 @@
 #include "soc_miwu.h"
 #include "soc_power.h"
 
+#define LOG_LEVEL CONFIG_UART_LOG_LEVEL
 #include <logging/log.h>
 LOG_MODULE_REGISTER(uart_npcx, LOG_LEVEL_ERR);
```

Comment:
Should we remove the LOG_LEVEL_ERR here?

Reply:
Yes.